content/en/mobile/_index.md

```diff
@@ -116,21 +207,22 @@ View your [monitors][14] from your home screen with Datadog widgets. Tap on any
 - Tap the “+” button on the top left corner of the screen.
 - Search for "Datadog" widgets.
 - Select your prefered size (small shows two monitor saved views, medium allows up to three monitor saved views, and large up to six monitor saved views).
-- Place the widget in your desired, onscreen location.
+- Drag the widget to your desired, on-screen location.
 
 
 {{% /tab %}}
 {{% tab "Android" %}}
 
-- Open the Android Widget Picker.
-- Either long press your home screen, or long press the Datadog app icon.
-- Tap on "Widgets". If you have app shortcuts, it might appear as just an icon on the top right corner of the bubble.
-- Drag the Monitor Saved Views widget on your home screen.
+- Long press on your home screen.
+- Tap the "Widgets" button on your home screen editor. If you have app shortcuts, it might appear as just an icon on the top right corner of the bubble.
```

Comment:
```suggestion
- Tap "Widgets" on your home screen editor. If you have app shortcuts, it might appear as just an icon on the top right corner of the bubble.
```